.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ commands:
           name: Install libssl-dev for openssl-sys
           command: sudo apt install -y libssl-dev
       - run:
-          name: Install libclang for rocksdb
+          name: Install libclang
```

Comment:
Can't we just remove these installs entirely if they're no longer needed for `rocksdb`

Reply:
`c-kzg` which is a dependency of `reth-primitives` requires it